services/api-server/tests/unit/conftest.py

```diff
@@ -107,9 +108,9 @@ async def client(app: FastAPI) -> AsyncIterator[httpx.AsyncClient]:
 
     # LifespanManager will trigger app's startup&shutown event handlers
     async with LifespanManager(app), httpx.AsyncClient(
-        app=app,
         base_url="http://api.testserver.io",
         headers={"Content-Type": "application/json"},
+        transport=ASGITransport(app=app),
```

Comment:
Out of curiosity, why this change? 🤔

Reply:
This fixes a deprecation warning. I added that in the description above.

Reply:
Aha, cool. Thanks a lot for the clarification